### PR TITLE
Create Footer Component - MariaD 11.27.24

### LINKED ISSUE #15: Create Footer component

### PR DESCRIPTION
**Pull request by tobemari**
closes #15 
This PR introduces a Footer component to centralize styling across the app.
Centralizing styling simplifies future design updates.

Key changes:
- Added a footer for branding and user engagement.
- Pushed all changes including HomePage and Header for more convenient review. (HomePage and Header are drafts for now)

### CODE REVIEW

File: src/Theme.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    text: { primary: "#FFFFFF", secondary: "#B3B3B3" },
+  },
+  typography: {
+    fontFamily: "'Inter', sans-serif",
+    h1: { fontSize: "32px", fontWeight: 400 },
+    body1: { fontSize: "24px", fontWeight: 400 },
+  },
+});
+
+export default theme;
```
Comment (umavenki) [inline src/Theme.jsx]: Theme looks good Maria. I see the same in PR https://github.com/Code-the-Dream-School/hh-team1-front/pull/17. Are you planning to merge Theme with this PR or the other one?

**Review by StuvaScript — CHANGES_REQUESTED** [review]
All of this code looks great!!

One request though. Can you push all the files that you've updated please. I had to do some digging around the internet to find and download all the dependencies and get the ThemeProvider component working to use the theme and check out the Footer component. That way we can have an easy merge to see all your good work :)

**Review by StuvaScript — APPROVED** [review]

**Comment by StuvaScript** [comment]
Oh wow, lots of great work! The only thing I had an issue with was in the Theme.jsx file I couldn't find the origin of the "@fontSource/Inter/400.css" import. But other than that this looks amazing!

**Comment by tobemari** [comment]
> Oh wow, lots of great work! The only thing I had an issue with was in the Theme.jsx file I couldn't find the origin of the "@fontSource/Inter/400.css" import. But other than that this looks amazing!

Hey Stu, you can find the `fontsource/inter` package in package.json.

**Review by jgoldenberg29 — APPROVED** [review]
Layout looks good to me! way to put in the work Mariia. There will most likely be some merge conflicts between all 4 of you especially in the app.jsx so make sure you all communicate about which code should remain.

**Comment by tobemari** [comment]
> Layout looks good to me! way to put in the work Mariia. There will most likely be some merge conflicts between all 4 of you especially in the app.jsx so make sure you all communicate about which code should remain.

Thank you Josh! I'll do the communication

**Pull request merged**